### ISSUE TITLE
Navigation - Multiple Methods

### ISSUE THREAD

**Issue by jinky32** [ThreeUK/digital-library.three.co.uk#5]
There should be more than one way for users to move around the site.

**Why is this required?** Some methods of navigation will suit some people better than others. For example, people with cognitive impairments may prefer to work with a site map which provides an overview of the website.

**Recommendation** a site map should be provided as well as a navigation menu.

**Exceptions**: Where web pages are generated as the result of, or a step in, a process, such as a payment process which you would not necessarily want to include in a site map.

**WCAG Reference**: This corresponds to WCAG 2.4.5 http://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-mult-loc.html
